recipe/build.sh

```diff
@@ -4,4 +4,10 @@ if [[ `uname` == 'Darwin' ]]; then
     export MACOSX_DEPLOYMENT_TARGET=10.9
 fi
 
+# Use only 1 thread with OpenBLAS to avoid timeouts on CIs.
+# This should have no other affect on the build. A user
+# should still be able to set this (or not) to a different
+# value at run-time to get the expected amount of parallelism.
+export OPENBLAS_NUM_THREADS=1
```

Comment:
As a safety measure, we are restricting it during the build too. Though it hasn't caused us problems yet. Figure it is better to be safe. Again this will not affect the end users ability to change this whatsoever.